examples/examples_test.py

```diff
@@ -93,3 +94,30 @@ def test_example_runs_grover():
 
 def test_example_runs_phase_estimator():
     examples.phase_estimator.main()
+
+
+def test_random_circuits_v2():
+    circuit = examples.random_circuits_v2.generate_random_v2_circuit_grid(
+        n_rows=4, n_cols=5, cz_depth=9, seed=0)
+    # We check that is exactly circuit inst_4x5_10_0
+    # in github.com/sboixo/GRCS cz_v2
+    assert len(circuit) == 11
+    assert sum( 1 for _ in circuit.findall_operations_with_gate_type(
+        cirq.ops.common_gates.Rot11Gate)) == 35
+    assert sum( 1 for _ in circuit.findall_operations_with_gate_type(
+        cirq.ops.common_gates.RotXGate)) == 15
```

Comment:
Don't dig down when not needed; use e.g. `cirq.RotXGate` instead of `cirq.bla.bla.RotXGate`.

If moving to `experiments` you would need to `from cirq import ops` and use `ops.RotXGate` to avoid cyclic dependency issues.